lib/helper/desktopHelper.py

```diff
@@ -35,6 +35,19 @@ def launch_browser(browser_type, input_profile_path=None):
     else:
         browser_obj = BrowserChrome(DEFAULT_BROWSER_HEIGHT, DEFAULT_BROWSER_WIDTH, profile_path)
     browser_obj.launch()
+    return profile_path
+
+
+def trigger_network_console(browser_type, sikuli_obj, input_env, profiler_list):
+    for profiler_data in profiler_list:
+        if profiler_data['name'] == input_env.PROFILE_NAME_HAR_PROFILER:
+            if browser_type == DEFAULT_BROWSER_TYPE_FIREFOX:
+                sikuli_obj.run(input_env.sikuli_path, input_env.hasal_dir, "test_firefox_trigger_network",
+                               input_env.test_method_name + "_" + input_env.time_stamp)
```

Comment:
this should be changed to use file name specified in environment (as we discussed today in the afternoon)